docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,188 @@
+If this is the case, you can pass a list of metrics or a dict in the format {metric name: metric} as a parameter to the
+check.
+
+The metrics on the list can be existing:
+
+*   `Ignite metrics <https://pytorch.org/ignite/metrics.html#complete-list-of-metrics>`__ for vision
+*   `Sklearn scorers <https://scikit-learn.org/stable/modules/model_evaluation.html>`__ for tabular
+*   strings from Deepchecks' `supported strings <#list-of-supported-strings>`__ for both vision and tabular
+
+or your own implementations.
+
+.. code-block:: python
+
+   train_ds, test_ds = adult.load_data(data_format='Dataset', as_train_test=True)
+   model = adult.load_fitted_model()
+
+   scorer = ['precision_per_class', 'recall_per_class', 'fnr_macro']
+   check = TrainTestPerformance(scorers=scorer)
+   result = check.run(train_ds, test_ds, model)
+   result
+
+
+List of Supported Strings
+=========================
+.. list-table:: Supported Strings
+   :widths: 25 75
+   :header-rows: 1
+
+   * - String
+     - Metric
+   * - 'neg_rmse'
+     - negative root mean squared error (Sklearn)
+   * - 'neg_mae'
+     - negative mean absolute error (Sklearn)
+   * - 'precision_per_class'
+     - precision per class - no averaging (results are returned per class)
+   * - 'recall_per_class' , 'sensitivity_per_class'
+     - recall per class - no averaging
+   * - 'f1_per_class'
+     - F-1  per class - no averaging
+   * - 'fpr_per_class'
+     - False Positive Rate per class - no averaging
+   * - 'fpr'
+     - False Positive Rate - binary
+   * - 'fpr_macro'
+     - False Positive Rate - macro averaging
+   * - 'fpr_micro'
+     - False Positive Rate - micro averaging
+   * - 'fpr_weighted'
+     - False Positive Rate - weighted macro averaging
+   * - 'fnr_per_class'
+     - False Negative Rate per class - no averaging
+   * - 'fnr'
+     - False Negative Rate - binary
+   * - 'fnr_macro'
+     - False Negative Rate - macro averaging
+   * - 'fnr_micro'
+     - False Negative Rate - micro averaging
+   * - 'fnr_weighted'
+     - False Negative Rate - weighted macro averaging
+   * - 'tnr_per_class'
+     - True Negative Rate per class - no averaging
+   * - 'tnr', 'specificity'
+     - True Negative Rate - binary
+   * - 'tnr_macro', 'specificity_macro'
+     - True Negative Rate - macro averaging
+   * - 'tnr_micro', 'specificity_micro'
+     - True Negative Rate - micro averaging
+   * - 'tnr_weighted', 'specificity_weighted'
+     - True Negative Rate - weighted macro averaging
+
+In addition to the strings listed above, all Sklearn
+`scorer strings
+<https://scikit-learn.org/stable/modules/model_evaluation.html#the-scoring-parameter-defining-model-evaluation-rules>`__
+apply.
+
+Custom metrics
+==============
+You can also pass your own custom metric to relevant checks and suites.
+
+Custom metrics should follow the
+`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API for computer vision or
```

Comment:
Perhaps this is the place to explain what scorers and ignite metrics are, rather than mentioning it in the "How can I use metrics with Deepchecks?" section (which I think should be removed)

What I mean is that in no place we explain that a scorer is a function with the signature f(model, x, y) rather than the more intuitive f(y_true, y_pred) that people probably expect for metrics.